workflow-engine/src/main/java/com/redhat/parodos/workflows/workflow/WorkFlowPropertiesMetadata.java

```diff
@@ -0,0 +1,18 @@
+package com.redhat.parodos.workflows.workflow;
+
+import lombok.Builder;
+
+@Builder
```

Comment:
What about generating getter and setter?

Reply:
> What about generating getter and setter?
@Data has getter and setter